src/actions/index.ts

```diff
@@ -0,0 +1,50 @@
+export enum ActionKeys {
+    LOAD = "LOAD",
+    LOADED = "LOADED",
+    TOUCH = "TOUCH",
+    TOUCHED = "TOUCHED",
+}
+
+export interface LoadAction {
+    type: ActionKeys.LOAD;
+    id: string;
+}
+
+export const load = (id: string): LoadAction => ({
+   type: ActionKeys.LOAD,
+   id,
+});
+
+export interface LoadedAction {
+    type: ActionKeys.LOADED;
+    oppName: string;
+    touches: number;
+}
+
+export const loaded = (oppName: string, touches: number): LoadedAction => ({
+    type: ActionKeys.LOADED,
+    oppName,
+    touches,
+});
+
+export interface TouchAction {
+    type: ActionKeys.TOUCH;
+    id: string;
+    currentTouches: number;
```

Comment:
this is pretty much just a glorified setter!!! It would be more appropriate to let the reducer determine how many total touches there are.   You could either remove `currentTouches` or make it hold the number of touches you would like to add.

Reply:
looks like this comment actually belongs on the `TouchedAction`.  But same idea